gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -978,7 +978,7 @@ end
 -- @realm shared
 function SWEP:DryFire(setnext)
     if CLIENT and LocalPlayer() == self:GetOwner() then
-        self:EmitSound("Weapon_Pistol.Empty")
+        self:EmitSound(")weapons/pistol/pistol_empty.wav", 75, 100, 0.7, CHAN_VOICE)
```

Comment:
```suggestion
        self:EmitSound("weapons/pistol/pistol_empty.wav", 75, 100, 0.7, CHAN_VOICE)
```

Reply:
Interesting, let's still switch to `CHAN_ITEM` as @wgetJane suggested :) then this is good to go!

Thanks btw for upstreaming your fixes <3